pkgs/applications/graphics/freecad/default.nix

```
@@ -101,6 +101,7 @@ mkDerivation rec {
   ];
 
   cmakeFlags = [
+    "-D BUILD_FLAT_MESH:BOOL=ON"
```

```suggestion
    "-DBUILD_FLAT_MESH:BOOL=ON"
```
this should eliminate any potential quoting issues.